Add maximum resolution note to location accuracy notice

PockyBum522 opened the pull request:
I had to look up what the setting was allowed to be, so I figured I'd relay that information where it may be useful.

brianshea2 reviewed (commented):
This is great, thank you. The only thing I'd prefer to change before committing is putting this note just after the quote, since it's not actually part of the quote, and then the parentheses could be removed as well.

PockyBum522 commented:
Moved it to its own line, per instruction. As it is now slightly more of a standalone concept, I added a small amount of defining text. Hopefully this feels appropriate. Thanks!

brianshea2 commented on README.md:
```diff
@@ -22,7 +22,9 @@ If your node can be heard by another node already reporting to MQTT, that's it!
 Meshtastic has [made a change to their MQTT server](https://meshtastic.org/blog/recent-public-mqtt-broker-changes/):
 
-> Only position packets with imprecise location information [10-16 bits] will be passed to the topic, ensuring that sensitive data is not exposed.
+Only position packets with imprecise location information [10-16 bits] will be passed to the topic, ensuring that sensitive data is not exposed.
```
```suggestion
> Only position packets with imprecise location information [10-16 bits] will be passed to the topic, ensuring that sensitive data is not exposed.
```
let's keep this a quote

PockyBum522 commented:
My bad. I apparently hadn't had my coffee yet and didn't realize that was markdown.
